deployment/kubernetes/yaml/orderhistoryservice.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-orderhistoryservice
+spec:
+  selector:
+    svc: ftgo-orderhistoryservice
+  ports:
+  - port: 8083
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftgo-orderhistoryservice
+spec:
+  selector:
+    matchLabels:
+      svc: ftgo-orderhistoryservice
+  template:
+    metadata:
+      labels:
+        svc: ftgo-orderhistoryservice
+    spec:
+      containers:
+      - name: ftgo-orderhistoryservice
+        image: eventuateexamples/eventuate-tram-examples-customers-and-orders-order-history-service:latest
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: "512Mi"
+            cpu: "1000m"
+        ports:
+        - containerPort: 8080
+        env:
+          - name: SPRING_DATA_MONGODB_URI
+            value: mongodb://mongodb/customers_and_orders
+          - name: EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS
+            value: ftgo-kafka:9092
+          - name: EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING
+            value: ftgo-zookeeper:8121
+          - name: SPRING_SLEUTH_ENABLED
+            value: "true"
+          - name: SPRING_SLEUTH_SAMPLER_PROBABILITY
+            value: "1"
+          - name: SPRING_ZIPKIN_BASE_URL
+            value: http://zepkin:9411
+        livenessProbe:
+          httpGet:
+            path: /actuator

```

Comment:
The path for all Java services should be `/actuator/health`

Reply:
updated